### DISCUSSION TITLE
How to add an image

### DISCUSSION THREAD

**Discussion by toomy78** [ggml-org/llama.cpp#8561] [Q&A]
Hello ! 👋

I've made some changes to the "examples/server/public/index.html" file that launches when server-llama.exe is launched.
However, my modifications are still visible, except for my image additions.
Do you know how to go about adding them? I've tried looking at the various CMakeLists files, but I can't find anything that doesn't send or link to the images in my 'img' folder.

Thank you for your answers

**Comment by toomy78** [comment]
Hello,

Any ideas please ?

Sorry for my raise

**Comment by dspasyuk** [comment]
@toomy78 Are you trying to submit a pull request? https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project

**Reply by toomy78** [reply]
Yes I've already done a "git pull"
But when I add an image here "examples/server/public/index.html" :
`<img src="img/llamacpp.png" width="150" />`
This image is not accessible at my url  "http://126.0.0.1:8080"

**Reply by dspasyuk** [reply]
I am not sure about your specific case but in general, open dev tools in Firefox or Chrome, go to network check if the image is getting loaded in the page, make sure to reload the page when you in network tab. Also I do not think you need slash at the end of the img tag. Also, I would I assume only images stored in a public folder will be loaded, so make sure your image is place in the right place.

**Reply by toomy78** [reply]
Yes, that's my problem. I don't know what to do now after the update. 
It doesn't seem that when I order :
cmake -B build
cmake --build build --config Release
It won't retrieve my added files
But thanks a lot for your answer, I'll try to find out what's wrong with it ;)
